app/controllers/publishers_controller.rb

```diff
@@ -59,7 +59,7 @@ def create
 
     @publisher = Publisher.new(pending_email: params[:email])
 
-    @should_throttle = should_throttle_create?
+    @should_throttle = should_throttle_create? || params[:captcha] == "true"
```

Comment:
i think we don't need to check for "true", truthiness is sufficient (e.g. /?captcha=1)